commands/FBDebugCommands.py

```diff
@@ -168,7 +168,9 @@ def run(self, arguments, options):
 
         methodIsClassMethod = methodTypeCharacter == "+"
 
-        if not methodIsClassMethod:
+        if methodIsClassMethod:
+            methodTypeCharacter = "\+"
```

Comment:
I think later on on L245, there's a place where we use the method type character in a non-regex context. Perhaps we can have a separate variable for the original methodTypeCharacter, as well as an escaped one we use when constructing regular expressions?

Reply:
As it turns out, I commented this but didn't share it with my colleague who approved the change. If you'd like to submit a follow-on, that'd be great. If not, I can take care of it eventually. :)